sdk/sdk-view/src/test/java/mil/emp3/api/CameraLookAtTest.java

```diff
@@ -61,6 +61,12 @@ public void cameraTests() {
         validTiltCameraTest(global.CAMERA_TILT_TO_SKY);
         validHeadingCameraTest(global.HEADING_MINIMUM);
         validHeadingCameraTest(global.HEADING_MAXIMUM);
+        camera.apply(false);
```

Comment:
what is this trying to test?

Reply:
In the original test, I noticed that apply was never called, so effectively the changes to settings were never processed.  The apply boolean parameter requests animation.  I have two calls, one with false, one with true (i.e. without and with animation).  The second parameter, when used, is the user context.